src/templates/cda/Sections/CarePlan.hbs

```diff
@@ -0,0 +1,39 @@
+{{#if (contains (toString (toJsonString msg)) '2.16.840.1.113883.10.20.22.2.10')}}
+  {{#with (getFirstCdaSectionsByTemplateId msg '2.16.840.1.113883.10.20.22.2.10') as |section|}}
+    {{#with (generateUUID (toJsonString section)) as |Id| }}
+      {
+        "fullUrl":"urn:uuid:{{Id}}",
+        "resource":{
+            "resourceType": "CarePlan",
+            "id":"{{Id}}",
+            "text": {
+              "status": "additional", {{!-- Specifying this additional because most of the structured data below is missing  --}}
+              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">{{section.2_16_840_1_113883_10_20_22_2_10.title._}}</div>"
```

Comment:
I think you want to be pulling from the text fields, not the title, i.e. the items under section.2_16_840_1_113883_10_20_22_2_10.text.list.item

Reply:
The challenge is the structure of the `text` field is ad hoc.  In our case it's a list but that's not consistent with other examples I've looked at.  Sometimes it's a HTML table or a div with content.  The ideal solution would be to extract the HTML from the `text` node but I couldn't figure out how to do within handlebars.

Reply:
I think it's fine to assume we'll only ever run this against chirp generated CCDAs and so tailor it to that. I wrote a helper in this pull (not merged yet) https://github.com/onemedical/FHIR-Converter/pull/20/files to extract the text by reference, though in this case I would imagine the thing to do is just grab all the text in that section and shove it in here.